modules/apps/batch-engine/batch-engine-test/src/testFunctional/macros/BatchEngine.macro

```diff
@@ -16,18 +16,27 @@ definition {
 		return ${archivedFilePath};
 	}
 
-	macro assertExportedJsontContainsCorrectConfigurationObject {
-		Variables.assertDefined(parameterList = "${jsonObject},${expectedValue}");
+	macro assertExportedFileContainsCorrectObject {
+		Variables.assertDefined(parameterList = "${fileName},${jsonObject},${expectedValue}");
 
-		var fileContent = TestCase.getTempFileContent(fileName = "export.batch-engine-data.json");
+		var fileContent = TestCase.getTempFileContent(fileName = ${fileName});
 
-		var actualValue = JSONUtil.getWithJSONPath(${fileContent}, "$.configuration.${jsonObject}");
+		var actualValue = JSONUtil.getWithJSONPath(${fileContent}, ${jsonObject});
 
 		TestUtils.assertEquals(
 			actual = ${actualValue},
 			expected = ${expectedValue});
 	}
 
+	macro assertExportedJsontContainsCorrectObject {
```

Comment:
Isn't this redundant? Couldn't we use assertExportedFileContainsCorrectObject directly to check the JsonT file?

Reply:
This is because the method assertExportedJsontContainsCorrectObject is already merged into Brian's master, but not on liferay/master yet, so that's why we are seeing changes on this and the following ExportOfJsontIncludeConfigurationObject.testcase